### ISSUE TITLE
document Local()

### ISSUE THREAD

**Issue by collinanderson** [django/asgiref#256]
Hi All,

Currently there's a hint in the readme that asgiref provides thread locals, but they're undocumented as far as I can tell.

Thanks!
Collin

**Comment by andrewgodwin** [comment]
Ah yes, excellent point. They are _mostly_ drop-in for `threading.Local`, but it is likely wise to explain the differences.

**Comment by collinanderson** [comment]
but _what_ is a drop in replacement? where do I import it from?

**Comment by andrewgodwin** [comment]
Basically, anywhere you would write:
```
from threading import Local
```
you can instead write
```
from asgiref.local import Local
``` and it will work the same, with the same API, except it will differentiate based on coroutine rather than just by thread.

**Comment by collinanderson** [comment]
Right, yeah exactly. What you just said should probably be added to the readme and/or documentation.
